<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户控制台</string>
  </property>
  <property name="fixedSize" stdset="0">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topButtonLayout">
      <item><widget class="QPushButton" name="networkMonitorBtn"><property name="text"><string>网络监控</string></property><property name="icon"><iconset resource="resources/img/img.qrc"><normaloff>:/img/monitorcenter.png</normaloff></iconset></property><property name="styleSheet"><string notr="true">QPushButton { background-color: #2196F3; color: white; border-radius: 10px; padding: 10px 28px; font-size: 18px; font-weight: bold; margin: 0 12px; min-width: 120px; border: none; } QPushButton:hover, QPushButton:checked { background-color: #1565C0; box-shadow: 0 2px 12px #90caf9; }</string></property></widget></item>
      <item><widget class="QPushButton" name="personalSettingsBtn"><property name="text"><string>个人设置</string></property><property name="icon"><iconset resource="resources/img/img.qrc"><normaloff>:/img/user.png</normaloff></iconset></property><property name="styleSheet"><string notr="true">QPushButton { background-color: #2196F3; color: white; border-radius: 10px; padding: 10px 28px; font-size: 18px; font-weight: bold; margin: 0 12px; min-width: 120px; border: none; } QPushButton:hover, QPushButton:checked { background-color: #1565C0; box-shadow: 0 2px 12px #90caf9; }</string></property></widget></item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <widget class="QWidget" name="welcomePage">
       <layout class="QVBoxLayout" name="welcomeLayout">
        <item>
         <widget class="QLabel" name="welcomeLabel">
          <property name="text">
           <string>欢迎使用用户控制台！请选择上方功能进行操作。</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimumWidth">
           <number>600</number>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 32px; font-weight: 600; color: #1565C0; margin: 100px 0 60px 0; letter-spacing: 2px; text-shadow: 1px 2px 8px #e3eaf7; border: none; background: transparent;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomButtonLayout">
      <item><spacer name="bottomSpacer"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer></item>
      <item><widget class="QPushButton" name="logoutBtn"><property name="text"><string>退出登录</string></property><property name="styleSheet"><string notr="true">QPushButton { background-color: #90A4AE; color: white; border-radius: 8px; padding: 8px 20px; font-size: 14px; font-weight: bold; margin: 0 8px;} QPushButton:hover { background-color: #607D8B; }</string></property></widget></item>
      <item><widget class="QPushButton" name="closeBtn"><property name="text"><string>关闭程序</string></property><property name="styleSheet"><string notr="true">QPushButton { background-color: #E57373; color: white; border-radius: 8px; padding: 8px 20px; font-size: 14px; font-weight: bold; margin: 0 8px;} QPushButton:hover { background-color: #C62828; }</string></property></widget></item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_logout"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_function">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="action_monitor"/>
    <addaction name="action_settings"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_function"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_logout">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="action_monitor">
   <property name="text">
    <string>数据监控</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>个人设置</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/img/img.qrc"/>
 </resources>
 <connections/>
</ui>
